From 1abd458351a44c14942b93ca77084a7cc0e3589e Mon Sep 17 00:00:00 2001
From: Xiretza <xiretza@xiretza.xyz>
Date: Thu, 25 Nov 2021 15:59:16 +0100
Subject: [PATCH 1/2] Remove misguided attempt at reproducibility that breaks
 debug package

makepkg uses -fdebug-prefix-map=$srcdir=/usr/src/debug to make debug
packages work, -ffile-prefix-map overrides this.
---
 src/CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 4aa755a4..57bd0fae 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -252,7 +252,6 @@ add_library(Catch2 STATIC
   ${BENCHMARK_HEADERS}
   ${BENCHMARK_SOURCES}
 )
-add_build_reproducibility_settings(Catch2)
 add_library(Catch2::Catch2 ALIAS Catch2)
 
 if (ANDROID)
@@ -302,7 +301,6 @@ target_include_directories(Catch2
 add_library(Catch2WithMain STATIC
     ${SOURCES_DIR}/internal/catch_main.cpp
 )
-add_build_reproducibility_settings(Catch2WithMain)
 add_library(Catch2::Catch2WithMain ALIAS Catch2WithMain)
 target_link_libraries(Catch2WithMain PUBLIC Catch2)
 set_target_properties(Catch2WithMain
-- 
2.35.1

